[HIPIFY] Issue running hipify-clang  - unit32_t

I am having an issue with running hipify-clang.  I have cloned the latest and built, downloaded and install CUDA 10.2, and downloaded LLVM+Clang 10.0.
First I get the warning :
warning: Unknown CUDA version 10.2. Assuming the latest supported version 10.1

I thought it was supposed to be compatible with 10.2?
Next when it tries to run and I am using the square.cu example, I get:

In file included from <built-in>:1:
In file included from /home/drew/hip/HIP/hipify-clang/dist/include/__clang_cuda_runtime_wrapper.h:48:
/usr/local/cuda-10.2/include/cuda.h:59:9: error: unknown type name 'uint32_t'
typedef uint32_t cuuint32_t;

Any ideas?

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -127,10 +127,14 @@ void appendArgumentsAdjusters(ct::RefactoringTool &Tool, const std::string &sSou
   std::string clang_inc_path = std::string(llvm::sys::path::parent_path(hipify));
   clang_inc_path.append("/include");
   Tool.appendArgumentsAdjuster(ct::getInsertArgumentAdjuster(clang_inc_path.c_str(), ct::ArgumentInsertPosition::BEGIN));
-  Tool.appendArgumentsAdjuster(ct::getInsertArgumentAdjuster("-isystem", ct::ArgumentInsertPosition::BEGIN));
+  Tool.appendArgumentsAdjuster(ct::getInsertArgumentAdjuster("-Xclang", ct::ArgumentInsertPosition::BEGIN));
+  Tool.appendArgumentsAdjuster(ct::getInsertArgumentAdjuster("-internal-isystem", ct::ArgumentInsertPosition::BEGIN));
+  Tool.appendArgumentsAdjuster(ct::getInsertArgumentAdjuster("-Xclang", ct::ArgumentInsertPosition::BEGIN));
   clang_inc_path.append("/cuda_wrappers");
   Tool.appendArgumentsAdjuster(ct::getInsertArgumentAdjuster(clang_inc_path.c_str(), ct::ArgumentInsertPosition::BEGIN));
-  Tool.appendArgumentsAdjuster(ct::getInsertArgumentAdjuster("-isystem", ct::ArgumentInsertPosition::BEGIN));
+  Tool.appendArgumentsAdjuster(ct::getInsertArgumentAdjuster("-Xclang", ct::ArgumentInsertPosition::BEGIN));
+  Tool.appendArgumentsAdjuster(ct::getInsertArgumentAdjuster("-internal-isystem", ct::ArgumentInsertPosition::BEGIN));
+  Tool.appendArgumentsAdjuster(ct::getInsertArgumentAdjuster("-Xclang", ct::ArgumentInsertPosition::BEGIN));
   // Ensure at least c++11 is used.
   std::string stdCpp = "-std=c++11";
 #if defined(_MSC_VER)

